models/voters.py

```diff
@@ -30,11 +32,18 @@ class VoteRecord(db.Model):
 
     @staticmethod
     def lookup(last, first, middle, city, county, limit):
-        return (VoteRecord.query.filter(VoteRecord.first.like(f"{first}%"))
+        return (
+            VoteRecord.query.filter(VoteRecord.first.like(f"{first}%"))
             .filter(VoteRecord.last == last)
             .filter(VoteRecord.middle.like(f"{middle}%"))
             .filter(VoteRecord.county.like(f"{county}%"))
             .filter(VoteRecord.city.like(f"{city}%"))
+            .filter(
+                or_(
+                    VoteRecord.special_app_status.isnot(None),
+                    VoteRecord.special_ballot_status.isnot(None),
+                )
```

Comment:
i stumbled across this change and i'm wondering if we really want to do this. specifically what this is doing is filtering out real voters who did not apply to vote early in the special election. it seems useful if people can find that such a voter exists, so they know they typed the name correctly and that they're looking at the right person. e.g. if i search for my friend "Ben Lorber" and I see that there's a Ben Lorber who voted, if that turns out to be the wrong Ben Lorber then I might have a false sense of completion.

thoughts on this? was the motivation here to reduce the frequency/confusion around scenarios such as a dead person who shows up or something? i wonder if we can resolve that by making the wording really clear for people in this scenario (obviously not the real words we'd use but we'd be communicating: "they did NOT yet vote in this election! maybe they're dead. or maybe you should ping them and get them to vote!")

Reply:
The motivation here is to have a clearer data set that we're searching -- before, someone would show up in the results if they have voted in the special election, or if they voted early in the general but not if they voted day-of in the general, or if they're registered. I think it would be really confusing if you searched for two of your friends, one of whom voted day-of and one of whom voted early, and only one of those searched returns results, and is going to make people think they're not registered to vote or stuff like that. I think it's easier to say "if they're not in the results, that means they haven't voted in the special election" rather than "if they're not in the results, that means they both haven't voted in the special election, AND they either voted day-of in the general, or didn't vote in the general, but we can't tell you which."

Reply:
That makes sense, thanks for explaining it to me.

I think this is something where if we had the full voter file we could more easily avoid the confusion. Should we look into that? cc @nickcatal